### ISSUE TITLE
Faucet interface: asset denoms incorrect

### ISSUE THREAD

**Issue by gavinly** [anoma/namada-interface#406]
it seems like all assets are assumed to have micro denoms (ie. 6 decimal places), but ETH has 18 and BTC has 8

if i request 1000 NAM, i get 1000 NAM
if i request 3 BTC, i get 0.03 BTC
if i request 15 ETH, i get 0.000000000015 ETH

**Comment by gavinly** [comment]
asking the faucet for 300 BTC gets me the 3 BTC that i wanted

**Comment by gavinly** [comment]
@Fraccaman

**Comment by Fraccaman** [comment]
yeah we forgot to take into account denomination for each token! the backend is also wrong. Will fix it :) thanks for filling the issue!

**Comment by mateuszjasiuk** [comment]
Not a frontend issue
